### ISSUE TITLE
Highlight themes which pass accessibility best practices

### ISSUE THREAD

**Issue by kmturley** [ajaxorg/ace#5848]
### Describe the feature

As a developer, I would like to select a theme which passes accessibility best practices.

### Use Case

Currently I have to toggle through every theme and run a Google Lighthouse report to see which pass/fail.

### Proposed Solution

- Run an accessibility report on every theme
- Categorize/group themes by their score/rating
- Highlight themes which pass accessibility best practices on the ace website

### Other Information

Example Google Lighthouse output for monokai:
<img width="717" height="649" alt="Image" src="https://github.com/user-attachments/assets/fe61c418-3911-4b61-a928-722cb42fb7e7" />

### Acknowledgements

- [x] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### ACE version used

latest

**Comment by marinsokol5** [comment]
Hey @kmturley, you can use https://github.com/ajaxorg/ace/blob/master/src/theme/cloud_editor.js and https://github.com/ajaxorg/ace/blob/master/src/theme/cloud_editor_dark.js, these 2 are accessible with a contrast satisfying WCAG 2.2 AA (so 4.5:1 contrast ratio).

**Comment by kmturley** [comment]
Thanks yes those both work also Tomorrow Night Bright.
But I think the point still stands, that it should be highlighted somewhere in the documentation.
